zh-CN/firefox_cloud_services_tos.md

```diff
@@ -1,95 +1,95 @@
-﻿# Firefox Cloud Services：服务条款
+﻿# Mozilla 帐户服务条款
 
-最后更新于 2022 年 4 月 10 日
-{: datetime="2022-04-10" }
+最后更新于 2023 年 11 月 1 日
+{: datetime="2023-11-01" }
 
 ## 摘要
 
 本文首先对下文条款进行了概括。本摘要旨在帮助您了解各项条款，但请务必阅读整个文件，因为当您同意时，即表示您接受所有条款，而不仅是接受本摘要。
 
-* Firefox Cloud Services（以下简称“Services”）是 Mozilla 向您提供的一套服务。
+* Mozilla 帐户（“服务”）包括您的帐户以及 Mozilla 使用该帐户向您提供的一整套服务。
 * 服务以“现状”提供，并不以任何形式提供担保。对于因您使用 Services 造成的任何损害，Mozilla 承担的责任极其有限。
 
 ## 服务条款
 
 1. #### 简介
 
-下列服务条款（以下简称“<u>条款</u>”）管辖您对 Firefox Cloud Services — Mozilla 提供的一套在线服务（以下简称“Services”）的使用。
+    下列服务条款（以下简称"<u>条款</u>"）管辖您对 Mozilla 帐户以及 Mozilla 提供的一套在线服务（以下简称“Services”）的使用。
 
-2. #### Firefox 帐户
+2. #### Mozilla 帐户
 
-要使用某些服务，您将需要创建一个 Firefox 帐户。在注册过程中，系统将提示您设置密码。您有责任对自己的密码保密，并对通过 Firefox 帐户进行的活动负责。对于因您的 Firefox 帐户遭到未经授权使用而造成的任何损失，Mozilla 概不负责。
+     要使用某些服务，您将需要创建一个 Mozilla 帐户。在注册过程中，系统将提示您设置密码。您有责任对自己的密码保密，并对通过 Mozilla 帐户进行的活动负责。对于因您的 Mozilla 帐户遭到未经授权使用而造成的任何损失，Mozilla 概不负责。
 
 3. #### 服务
 
-(a) Mozilla VPN 是与 [Mullvad](https://mullvad.net/) 联合推出的虚拟专用网络。必需具有 Firefox 帐户。
+    (a) Mozilla VPN 是与 [Mullvad](https://mullvad.net/) 联合推出的虚拟专用网络。必需具有 Mozilla 帐户。
     
-(b) Firefox Relay 允许生成唯一别名电子邮件地址，让您在不泄露主要电子邮件地址的情况下收发电子邮件，从而保障主要电子邮件地址安全和隐私，免受在线服务的干扰。必需具有 Firefox 帐户。
+    (b) Firefox Relay 允许生成唯一别名电子邮件地址，让您在不泄露主要电子邮件地址的情况下收发电子邮件，从而保障主要电子邮件地址安全和隐私，免受在线服务的干扰。必需具有 Mozilla 帐户。
     
-(c) Firefox Monitor 是一项信息服务，它会在发生公共数据泄露时发出通知，从而加强在线安全性。您可以扫描我们网站中的电子邮件地址，了解大部分公开泄露事件。如果使用 Firefox 帐户订阅服务，您将会获得：
+    (c) Firefox Monitor 是一项信息服务，它会在发生公共数据泄露时发出通知，从而加强在线安全性。您可以扫描我们网站中的电子邮件地址，了解大部分公开泄露事件。如果使用 Mozilla 帐户订阅服务，您将会获得：
 
-* 完整报告：其中包括公开黑客的完整列表，包括各种敏感网站泄露，如儿童、成人和约会网站泄露事件（为保护上述人员的隐私，此类网站不在网站扫描范围之列）。
-* 泄露警报：如果您的电子邮件地址似乎受到最新数据泄露事件的波及，将向您发出通知。
-* 安全提示：间或向您发送 Mozilla 在线安全提示。
+    * 完整报告：其中包括公开黑客的完整列表，包括各种敏感网站泄露，如儿童、成人和约会网站泄露事件（为保护上述人员的隐私，此类网站不在网站扫描范围之列）。
+    * 泄露警报：如果您的电子邮件地址似乎受到最新数据泄露事件的波及，将向您发出通知。
+    * 安全提示：间或向您发送 Mozilla 在线安全提示。
 
-您有责任使用不同的强密码、安全问题、双因素身份验证、密码管理器及其他安全措施保护个人在线帐户。虽然 Firefox Monitor 和 [Have I Been Pwned?](https://haveibeenpwned.com/) 竭力获取最新数据，但依然无法保证全面或及时地记录所有公开数据泄露。
+    您有责任使用不同的强密码、安全问题、双因素身份验证、密码管理器及其他安全措施保护个人在线帐户。虽然 Firefox Monitor 和 [Have I Been Pwned?](https://haveibeenpwned.com/) 竭力获取最新数据，但依然无法保证全面或及时地记录所有公开数据泄露。
 
-(d) Firefox Notes 允许您在 Firefox 中进行备注。必需具有 Firefox 帐户，才能查看、编辑及删除不同设备的备注。
+    (d) Firefox Notes 允许您在 Firefox 中做备注。必需具有 Mozilla 帐户，才能查看、编辑及删除不同设备的备注。
 
-(e) Firefox Sync 允许您跨设备访问存储在 Firefox 中的信息，包括您的选项卡、智能地址栏、密码、书签和浏览器首选项。您可以通过 Firefox 的“设置”部分启用和禁用 Firefox Sync。必需具有 Firefox 帐户。
+    (e) Firefox Sync 允许您跨设备访问存储在 Firefox 中的信息，包括您的选项卡、智能地址栏、密码、书签和浏览器首选项。您可以通过 Firefox 的“设置”部分启用和禁用 Firefox Sync。必需具有 Mozilla 帐户。
     
-(f) Pocket 可帮助您发现根据个人兴趣量身定制的精彩内容并保存这些内容，以便日后随时使用任意设备再次查看。
+    (f) Pocket 可帮助您发现根据个人兴趣量身定制的精彩内容并保存这些内容，以便日后随时使用任意设备再次查看。
 
 4. #### 隐私政策
 
-我们将根据 [Mozilla 隐私政策](https://www.mozilla.org/privacy/)的规定使用通过 Services 接收的信息。隐私声明更详细地描述了我们通过各项服务接收的数据：
+    我们将根据 [Mozilla 隐私政策](https://www.mozilla.org/privacy/)的规定使用通过 Services 接收的信息。隐私声明更详细地描述了我们通过各项服务接收的数据：
```

Comment:
The links here were not parsed correctly because of the incorrect indentation. Fixed in https://github.com/mozilla/legal-docs/pull/1957/commits/899b1c714159da29e5a198cfff38bfe36b848734